include/EventLoop.h

```diff
@@ -94,5 +93,6 @@ class EventLoop
      * however care must be taken to typecast it correctly at the receiver end before dereferencing to prevent incorrect interpretation or 
      * segmentation faults in case of invalid or NULL pointers
      */
-    static void TriggerEvent(const std::string& evtName, const size_t& timeoutMS, void* data = nullptr);
-};
+    void TriggerEvent(const std::string& evtName, const size_t& timeoutMS, void* data = nullptr);
+
+}//namespace EventLoop
```

Comment:
leave a space after closing brace and // to make it look a bit cleaner. Make this change wherever applicable 
`} // namespace EventLoop`